app/src/main/java/org/fossasia/phimpme/gallery/activities/SplashScreen.java

```diff
@@ -40,13 +42,13 @@
     public final static int ALBUMS_PREFETCHED = 23;
     public final static int PHOTOS_PREFETCHED = 2;
     public final static int ALBUMS_BACKUP = 60;
-    private boolean PICK_INTENT = false;
+    private static boolean PICK_INTENT = false;
     public final static String ACTION_OPEN_ALBUM = "vn.mbm.phimp.leafpic.OPEN_ALBUM";
 
     //private HandlingAlbums albums;
-    private Album album;
-    private boolean can_be_finished = false;
-    private Intent nextIntent = null;
+    private static Album album;
+    private static boolean can_be_finished = false;
```

Comment:
Reference them from the context in the AsyncTask. That way you do not have to convert them to static.

Reply:
Okay sir. I am making changes now.

Reply:
Done @yashLadha  sir. I have made the requested changes in #1984